src/modules/main.js

```diff
@@ -0,0 +1,49 @@
+const inputBox = document.getElementById('input-box');
+const listContainer = document.getElementById('list-container');
+
+// localStorage
+function saveData() {
+  localStorage.setItem('data', listContainer.innerHTML);
+}
+
+function showList() {
+  listContainer.innerHTML = localStorage.getItem('data');
+}
+showList();
```

Comment:
- [ ] Please note that the `format` for storing user `data` in `localStorage` should be a string `key-value` pair, where the `key` is a `string` representing the data name, and the `value` is the data itself, which can be converted into a string using the `JSON.stringify()` method. Example of how you can store data in the `localStorage` is below:
```javascript
localStorage.setItem('username', JSON.stringify('LethaboMot'));
``` 

 - You're not to store the whole HTML in the local storage as you did: 
<img width="451" alt="Screenshot 2023-05-11 at 23 21 59" src="https://github.com/LethaboMot/weppack2/assets/107192076/4de6ac06-0ccd-4b20-94a5-3998d12196fa">

- [ ] Also please make sure to implement the changes the previous reviewer requested [here](https://github.com/LethaboMot/weppack2/pull/3/files#r1191672612) as I was unable to check that because your `localStorage` isn't setup properly.